### PR TITLE
Refactor: Make responsive community section

### LINKED ISSUE #1671: [refactor]: make responsive community section
### Is there an existing feature request for this?

- [X] I have searched the existing issues

### Summary

The current community section is inspired from k6, however, due to a lack of responsiveness the cards doesn't appear on the outer circular lines. Also the bunny image should be visible (small size) in the sm screen size at the top of the cards. 

We should also add hover animation on cards and parallax animation on the bunny image.

### Why should this be worked on?

for better UX

### Repository

website

### PR DESCRIPTION
**Pull request by hanzili**
fix [#1674](https://github.com/keploy/keploy/issues/1671)
1. make cards appear on the outer circular lines
image

2. Bunny image is now visible (small size) in the sm screen size at the top of the cards
image

3. add hover animation on cards
4. wasn't able to add parallax animation on the bunny image - the parallax effect requires changing the css properties of the bunny image based on scroll events. It requires additional logic to calculate the image's position dynamically, which easily conflict with the existing css for positioning cards around the bunny.

### CODE REVIEW

File: components/community.tsx [hunk 1]
```diff
@@ -1,459 +1,159 @@
...
-                <h2 className="text-lg font-semibold sm:text-md">Slack</h2>
-                <p className="text-gray-600 mt-1 md:text-sm sm:text-sm ">
-                  Connect and chat with other Keploy users
-                </p>
-              </div>
-            </a>
-          </div>
+type CardData = {
+  link: string;
+  svgIcon: any;
```
Comment (Hermione2408) [inline components/community.tsx]: please avoid using any it's not a good practice

**Review by Hermione2408 — CHANGES_REQUESTED** [review]

File: components/community.tsx (hunk 1 above)
Comment (hanzili) [reply components/community.tsx]: Agree! Since `svgIcon` is always going to be used as a url, I simply type it as a `string`.

**Review by Hermione2408 — APPROVED** [review]
LGTM @nehagup

**Review by nehagup — APPROVED** [review]
LGTM! 🫰

**Pull request merged**